decidim-core/spec/services/decidim/log_reference_generator_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::LogReferenceGenerator do
+  let(:generator) { described_class.new(request) }
+
+  let(:request) do
+    ActionDispatch::Request.new(request_env).tap do |request|
+      request.request_method = "GET"
+      request.request_id = SecureRandom.uuid
+    end
+  end
+  let(:request_env) { {} }
+  let(:organization) { create :organization }
```

Comment:
Unnecessary assignment for this spec, you are not using this anywhere in the spec.
```suggestion
```